com.unity.ml-agents/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Major Changes
+ - Introduced the `SideChannelUtils` to register, unregister and access side channels.
```

Comment:
Can you update the "Added a feature to allow sending stats..." below too? It still references Academy.Instance. GetSideChannel